docs/pages/management/admin/labels.mdx

```diff
@@ -13,18 +13,20 @@ the following:
 This guide demonstrates how to add labels to enrolled server resources.
 However, you can follow similar steps to add labels to other types of resources.
 
-## Static and dynamic labels
+## Static, dynamic, and ephemeral labels
 
-The labels you assign to resources can be **static labels** or **dynamic labels**.
+The labels you assign to resources can be **static labels**, **dynamic labels**, or **ephemeral labels**.
 
 - Static labels are hardcoded in the Teleport configuration file and don't change 
 while the `teleport` process is running. For example, you might use a static label to identify 
 the resources in a `staging` or `production` environment.
 - Dynamic labels—also known as **commands-based labels**—allow you to generate labels at runtime. 
 With a dynamic label, the `teleport` process executes an external command on its host at 
 a configurable frequency and the output of the command becomes the label value.
+- Ephemeral labels allow you to add labels to an instance without restarting the `teleport`
```

Comment:
I think calling them ephemeral is confusing. They're not supposed to be ephemeral, they're applied to every server heartbeat, right?

Can we not introduce any new names for them but just have a section explaining "this is how you can set node labels as a resource or via API instead of on the node"? Maybe call it something like "Setting node labels server-side".